dlp/dlp_snippets/triggers.go

```diff
@@ -69,6 +69,9 @@ func createTrigger(w io.Writer, client *dlp.Client, project string, minLikelihoo
 								},
 							},
 						},
+						TimespanConfig: &dlppb.StorageConfig_TimespanConfig{
```

Comment:
Would you mind adding a comment above this line with a short explanation of `TimespanConfig`? It's not immediately clear from the name.

Reply:
I do not think the idea here is to provide documentation on each field, but on how to perform an operation.
How about a link to the proto doc?
https://cloud.google.com/dlp/docs/reference/rest/v2/InspectJobConfig#TimespanConfig

Reply:
The goal is to provide a sample of the right way to call the API. Since it's not clear from the name/context what this field does, I think it would be helpful to have a short comment.

This will end up on https://cloud.google.com/dlp/docs/creating-job-triggers, which explains the field at the bottom of the page, but it would be confusing if you're reading from top to bottom.

Reply:
I am adding code samples for the enable_auto_population_of_timespan_config field.
The code snippet at the bottom will be replaced with pointers to this code once all languages are covered.
I added a comment to the field.

Reply:
Unfortunately, this also shows up in the "Create a job trigger" section. So, we probably need a separate sample function or a comment (something like `Optionally limit scans to only new content. See [link].`)